src/main/java/mate/academy/exception/RegistrationException.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.exception;
+
+public class RegistrationException extends Exception {
```

Comment:
```suggestion
public class RegistrationException extends RuntimeException  {
```

Reply:
Sorry, this suggestion was added by accident, in this task you should extends rom checked exception. The previous version of implementation was correct.